### ISSUE TITLE
Adding the word "Pointer" to "2.5.7 Dragging"

### ISSUE THREAD

**Issue by SuzanneTaylor** [w3c/wcag#1397]
This suggestion is to help reduce the possibility that those new to WCAG might mistake 2.5.7 as the full requirement for physical access to Dragging.

Using "2.5.7 Pointer Dragging" would help clarify this, help to relate this guideline to 2.5.1 and 2.5.2, and allow a search for "Pointer" bring up this set of guidelines.

To the same ends, in the Understanding document, 

"There is already a requirement for dragging features to be keyboard accessible, however, it is possible to create an interface that works with dragging and keyboard controls that does not work using only clicks or taps." 

could emphasize keyboard a bit more by adding references to those guidelines:

"2.1.1 Keyboard  and 2.1.3 Keyboard (No Exception) require dragging features to be keyboard accessible; however, it is possible to create an interface that works with dragging and keyboard controls that does not work using only clicks or taps."

**Comment by detlevhfischer** [comment]
Hi Suzanne Taylor,
I have updated the understanding text in this PR #1423  - it now has a section covering the distinction between pointer gestures and dragging movements.
I am not sure that changing the SC name to pointer dragging really helps though.
I have taken up your suggestion to directly refer to 2.1.1 / 2.1.3 when stating that dragging movements already need to be keyboard accessible.

**Comment by SuzanneTaylor** [comment]
I see what you are saying. Maybe something like "Click or Tap to Drag"?
It is a little long, but there are quite a few with "or" that are similar in length or longer:

- 1.2.3 Audio Description or Media Alternative (Prerecorded) 
- 1.4.7 Low or No Background Audio
- 1.4.13 Content on Hover or Focus
- 2.3.1 Three Flashes or Below Threshold

**Comment by detlevhfischer** [comment]
@SuzanneTaylor I don’t think as a title, something like “Click or tap to drag” will be making the intent much clearer - it might even be misinterpreted. We also have other success criteria like Keyboard, Orientation, Reflow etc. which do not explicate in what way that aspect needs to be covered....

**Comment by SuzanneTaylor** [comment]
I think Keyboard, Orientation and Reflow immediately bring to mind the basic realm of those guidelines, whereas Dragging is just the name of an interaction, similar to "Clicking", "Scrolling", "Closing Windows", "Viewing Images", "Hearing Audio" none of which have been used as titles for related guidelines.

**Comment by detlevhfischer** [comment]
@SuzanneTaylor fair point. I would then still prefer "Dragging Movements" because it chimes with "Pointer Gestures".
"Pointer Dragging" might suggest the pointer is dragged (moved across the screen) without dragging a target.

@alastc We could have a survey question and let the group choose the name.

**Comment by SuzanneTaylor** [comment]
I think "Dragging Movements" would resolve the issue nicely. It's clearly about movement, which leans strongly toward mouse or touch screen (rather than keyboard), and it avoids using just a commonly used term for a type of interaction, which could imply "read this to learn what to do about your drag and drop exercise". I like the chime with "Pointer Gestures" as well.

**Comment by alastc** [comment]
This was discussed and agreed yesterday: https://www.w3.org/2020/10/20-ag-minutes.html#item06

The change is here: https://github.com/w3c/wcag/pull/1484/files
